aptos-move/framework/src/error_map.rs

```diff
@@ -35,6 +35,7 @@ pub(crate) fn generate_error_map(
         test_mode: false,
         force_recompilation: false,
         fetch_deps_only: false,
+        fetch_latest_git_deps: false,
```

Comment:
We'll want to work through making this true in the CLI later.

Reply:
Yes that what I thought! Pining you once this landed.